xl2times/transforms.py

```diff
@@ -222,6 +222,44 @@ def discard(table):
     return result
 
 
+def revalidate_input_tables(
+    config: datatypes.Config,
+    tables: List[datatypes.EmbeddedXlTable],
+    model: datatypes.TimesModel,
+) -> List[datatypes.EmbeddedXlTable]:
+    """
+    Perform further validation of input tables by checking whether required columns are
+    present / non-empty. Remove tables without required columns or if they are empty.
+    """
+    result = []
+    for table in tables:
```

Comment:
This additional validation didn't fix my missing "uc_n" missing exception.

What _did_ work for austimes was to just drop empty dataframes, i.e. insert this block here
```python
        # drop empty dataframes
        if table.dataframe.empty:
            logger.warning(f"Dropping empty table {table.tag} in {table.filename}")
            continue
```

Reply:
🤣 That's a fun one! We actually drop all empty dataframes in `validate_input_tables`, but I guess at that point it is not yet empty. It probably has a comment row(s) that is dropped in `remove_comment_rows` afterwards.

Reply:
Yep, spot on - looks like that's exactly what is happening - there are tables where all the rows are commented (`\I:`) out.

Reply:
Opened #201 to handle this. @SamRWest any chance you could test?